packages/react-reconciler/src/__tests__/ReactSchedulerIntegration-test.js

```diff
@@ -1,580 +0,0 @@
-    expect(Scheduler).toHaveYielded(['Priority: Immediate']);
-  });
-
-  // TODO: Figure out what to do with these tests. I don't think most of them
-  // make sense once we decouple Scheduler from React. Perhaps need similar
-  // tests for React DOM.
-  // @gate !enableNativeEventPriorityInference
-  it('has correct priority during rendering', () => {
-    function ReadPriority() {
-      Scheduler.unstable_yieldValue(
-        'Priority: ' + getCurrentPriorityAsString(),
-      );
-      return null;
-    }
-    ReactNoop.render(<ReadPriority />);
-    expect(Scheduler).toFlushAndYield(['Priority: Normal']);
-
-    runWithPriority(UserBlockingPriority, () => {
-      ReactNoop.render(<ReadPriority />);
-    });
-    expect(Scheduler).toFlushAndYield(['Priority: UserBlocking']);
-
-    runWithPriority(IdlePriority, () => {
-      ReactNoop.render(<ReadPriority />);
-    });
-    expect(Scheduler).toFlushAndYield(['Priority: Idle']);
-  });
-
-  // TODO: Figure out what to do with these tests. I don't think most of them
-  // make sense once we decouple Scheduler from React. Perhaps need similar
-  // tests for React DOM.
-  // @gate !enableNativeEventPriorityInference
-  it('has correct priority when continuing a render after yielding', () => {
-    function ReadPriority() {
-      Scheduler.unstable_yieldValue(
-        'Priority: ' + getCurrentPriorityAsString(),
-      );
-      return null;
-    }
-
-    runWithPriority(UserBlockingPriority, () => {
-      ReactNoop.render(
-        <>
-          <ReadPriority />
-          <ReadPriority />
-          <ReadPriority />
-        </>,
-      );
-    });
-
-    // Render part of the tree
-    expect(Scheduler).toFlushAndYieldThrough(['Priority: UserBlocking']);
-
-    // Priority is set back to normal when yielding
-    expect(getCurrentPriorityAsString()).toEqual('Normal');
-
-    // Priority is restored to user-blocking when continuing
-    expect(Scheduler).toFlushAndYield([
-      'Priority: UserBlocking',
-      'Priority: UserBlocking',
-    ]);
-  });
-
-  it('layout effects have immediate priority', () => {
-    const {useLayoutEffect} = React;
-    function ReadPriority() {
-      Scheduler.unstable_yieldValue(
-        'Render priority: ' + getCurrentPriorityAsString(),
-      );
-      useLayoutEffect(() => {
-        Scheduler.unstable_yieldValue(
-          'Layout priority: ' + getCurrentPriorityAsString(),
-        );
-      });
-      return null;
-    }
-
-    ReactNoop.render(<ReadPriority />);
-    expect(Scheduler).toFlushAndYield([
-      'Render priority: Normal',
-      'Layout priority: Immediate',
-    ]);
-  });
-
-  // TODO: Figure out what to do with these tests. I don't think most of them
-  // make sense once we decouple Scheduler from React. Perhaps need similar
-  // tests for React DOM.
-  // @gate !enableNativeEventPriorityInference
-  it('passive effects never have higher than normal priority', async () => {
```

Comment:
We need to add an equivalent for this test, which fails until we address the `pendingPassiveEffectsRenderPriority`.